docs/users/plugins.md

```diff
@@ -128,4 +128,9 @@ formatted = mdformat.text(unformatted, extensions={"tables"})
     <td><code>deflist</code></td>
     <td>Adds support for <a href="https://pandoc.org/MANUAL.html#definition-lists">Pandoc-style</a> definition lists</td>
   </tr>
+  <tr>
+    <td><a href="https://github.com/KyleKing/mdformat-admon">mdformat-admon</a></td>
+    <td><code>admonitions</code></td>
```

Comment:
Should the rows in this table be sorted alphabetically or grouped?

Reply:
I think this used to be in alphabetical order but no longer is, so I don't care much. The order can be fixed seprately for all items if seen necessary.

Reply:
Thanks! I'm happy to make that change this evening, or you can go ahead and merge